.github/workflows/triage-labeled-issues.yml

```diff
@@ -0,0 +1,23 @@
+name: Triages labeled issues by adding them to a specific project field
+
+on:
+  issues:
+    types: [ labeled ]
+
+jobs:
+  move-ts-issues:
+    if: ${{contains(github.event.label.name, format('area{0} typescript', ':'))}}
+    uses: ./.github/workflows/move-issue-to-project-and-set-fields.yml
+    with:
+      field: 'Area'
+      field_option: '🟦 Typescript'
+      project_number: '65'
+    secrets: inherit
+  move-migration-issues:
+    if: ${{contains(github.event.label.name, format('area{0} migration ➡️', ':'))}}
```

Comment:
Not sure how that arrow is going to work but that's how the emoji pasted when I got it from the label 🤔

Reply:
Fwiw, it did show up correctly in my email. So _probably_ fine? 😁

Reply:
Ah interesting, I've found that some emojis render differently in vscode vs github so maybe there is some kind of mapping that happens. Learning a lot about emojis with this PR 😂